core/deployment/src/main/java/io/quarkus/deployment/logging/LogCleanupFilterBuildItem.java

```diff
@@ -11,8 +12,10 @@
  *
  * @author Stéphane Épardaud
  */
-public final class LogCleanupFilterBuildItem extends MultiBuildItem {
+public final class LogCleanupFilterBuildItem extends MultiBuildItem implements Comparable<LogCleanupFilterBuildItem> {
```

Comment:
This also should not need to be comparable.

Reply:
+1